packages/devtools_app/lib/src/screens/inspector/inspector_service.dart

```diff
@@ -1200,8 +1151,9 @@ class ObjectGroup extends ObjectGroupBase {
 
   Future<RemoteDiagnosticsNode?> getRootWidget() {
     return parseDiagnosticsNodeDaemon(
-      invokeServiceExtensionMethod(
-        RegistrableServiceExtension.getRootWidgetSummaryTreeWithPreviews,
+      invokeServiceMethodDaemonParams(
```

Comment:
all of the changes to use `WidgetInspectorServiceExtensions` are from https://github.com/flutter/devtools/pull/4633, so these should be removed from this PR